en/hardware/reference_design.md

```diff
@@ -31,10 +31,10 @@ The diagram below shows the division of bus and functional responsibilities betw
 Some Pixhawk-series controllers are built without the I/O board in order to reduce space or complexity, or to better address certain board use-cases.
 
 The I/O board is disabled by setting parameter [SYS_USE_IO=0](../advanced_config/parameter_reference.md#SYS_USE_IO).
-When the I/O board is disabled:
-- The MAIN mixer file is loaded into the FMU (so the "MAIN" outputs listed in the [Airframe Reference](../airframes/airframe_reference.md) appear on the port labeled AUX).
+When the I/O board is disabled: 
```

Comment:
@bkueng Can I get a PR from you to update this doc?

Reply:
This isn't relevant anymore and can be removed.
The mixing behavior won't change if you change `SYS_USE_IO`, except that you won't have the UI to configure the MAIN outputs anymore.
The behavior of the parameter changed now to not start the io driver if disabled. This means you won't have the MAIN outputs anymore, and also no RC if RC goes through the IO. I'm not sure if we even have to mention this here.

Reply:
@bkueng It is not quite as easy as that. The problem is that FCs seem to always label the PWM used by default by motor outputs as "MAIN" irrespective of whether it is actually the FMU/AUX. Agree the RC stuff is not so relevant.

I've corrected the text as I understood it. Please recheck.